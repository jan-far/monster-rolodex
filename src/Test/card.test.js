import React from "react";
import { shallow } from "enzyme";
import Cards from "../Components/Cards";

describe("Cards Tests", () => {
  const mockMonster = { id: 1, name: "ralph", email: "robo@gmail.com" };
  it("Make a snapshot of Cards comonent", () => {
    expect(shallow(<Cards monster={mockMonster} />)).toMatchSnapshot();
  });
});
