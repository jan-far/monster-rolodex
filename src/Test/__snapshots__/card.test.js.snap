// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards Tests Make a snapshot of Cards comonent 1`] = `ShallowWrapper {}`;
